packages/frontend/components/Buttons/ProfileButton/index.tsx

```diff
@@ -1,35 +1,27 @@
 import Image from 'next/image';
 import { useRouter } from 'next/router';
-import { useEffect } from 'react';
 import { useSelector } from 'react-redux';
-import { User } from '../../../models/User/user.model';
+import { isNftImage } from '../../../models/User/user.model';
+import { RootState } from '../../../store/store';
 
 
 type ProfileButtonProps = {
     disabled: boolean
 }
 
 export const ProfileButton = ({disabled}: ProfileButtonProps) => {
-    const user: User = useSelector((state: any) => state.user.selectedUser);
+    const user = useSelector((state: RootState) => state.user.selectedUser);
     const router = useRouter();
 
+    const profilePicture = user?.picture && !isNftImage(user.picture) ? user.picture.original.url : "/assets/icons/profile.svg"
+
     const handleClick = () => {
-        router.push(`/profile/${user.id}`);
+        router.push(`/profile`);
     }
-
-    useEffect(() => {
-        console.log(user)
-    }, [user])
-    
         
     return (
-        <button onClick={handleClick} disabled={disabled} className={"flex items-center bg-lime border-3 border-black border-solid rounded-full " + (disabled ? "opacity-30 " : "comic-border-mini ") + (user.name ? "" : "p-2") } type="button">
-            {
-                user?.coverPicture?.uri ?
-                <Image src={user?.coverPicture?.uri || "/assets/icons/profile.svg"} width="48" height="48" />
-                :
-                <Image src={user?.coverPicture?.uri || "/assets/icons/profile.svg"} width="24" height="24" />
-            }
+        <button onClick={handleClick} disabled={disabled} className={`overflow-hidden flex items-center justify-center bg-lime border-3 border-black border-solid rounded-full w-12 h-12 ${disabled ? 'opacity-30' : 'comic-border-mini'}`} type="button">

```

Comment:
accessibility wise we need to provide an alternative text as it's not clear what would happen when you click the button.
```
<button aria-label="Go to profile page" ...>
```
also `img` `alt` can be replaced with just an empty string as it has no contextual meaning to the user 
```
            <img src={profilePicture} alt="avatar" />
```